src/main/java/org/opentripplanner/standalone/server/DefaultServerRequestContext.java

```diff
@@ -41,9 +34,9 @@ public class DefaultServerContext implements OtpServerContext {
    * Copy constructor - used to make an HTTP Request scoped copy of the context. All mutable
    * components need to be coped here. This is
```

Comment:
incomplete comment

Reply:
Comment is no longer correct, I fixed it.